plugin-server/src/main/ingestion-queues/kafka-queue.ts

```diff
@@ -156,11 +55,9 @@ export class KafkaQueue implements Queue {
             status.info('⏬', `Connecting Kafka consumer to ${this.pluginsServer.KAFKA_HOSTS}...`)
             this.wasConsumerRan = true
 
-            const ingestionTopic = this.pluginsServer.KAFKA_CONSUMPTION_TOPIC!
-
-            await this.consumer.subscribe({
-                topic: ingestionTopic,
-            })
+            for (const topic of Object.keys(this.eachBatch)) {
```

Comment:
This is not a safe change as it will deploy the buffer live - transforms this from a safe refactor to an operational change.

Let's keep this as it was and only subscribe to the ingestion topic. I'll add the business logic here as part of the process split.